cli/docker_image/show.py

```diff
@@ -1,36 +1,103 @@
 #!/usr/bin/env python3
 
+import xml.etree.ElementTree as ET
+
+
 def configure_parser(docker_image_subparser):
-    docker_image_show = docker_image_subparser.add_parser('show', help="Show informations about a docker image")
-    docker_image_show_subparser = docker_image_show.add_subparsers(dest="information", help="Information to show")
+    docker_image_show = docker_image_subparser.add_parser(
+        "show", help="Show informations about a docker image"
+    )
+    docker_image_show_subparser = docker_image_show.add_subparsers(
+        dest="information", help="Information to show"
+    )
     docker_image_show_subparser.required = True
 
     init_show_vulnerabilities_subcommand(docker_image_show_subparser)
 
+
 def subcommand(args, api):
     if args.information == "vulnerabilities":
         vulnerabilities_subcommand(args, api)
     else:
         print(f"'{args.object}'")
 
+
 # Vulnerabilities
 
+
 def init_show_vulnerabilities_subcommand(docker_image_show_subparser):
-    docker_image_show_vulnerabilities_parser = docker_image_show_subparser.add_parser("vulnerabilities", help="Show vulnerabilities about a scanned docker image")
+    docker_image_show_vulnerabilities_parser = docker_image_show_subparser.add_parser(
+        "vulnerabilities", help="Show vulnerabilities about a scanned docker image"
+    )
     docker_image_show_vulnerabilities_parser.add_argument("docker_image_id")
+    docker_image_show_vulnerabilities_parser.add_argument(
+        "--format",
+        help="Output format. The only supported format is 'xml:filename.xml'.",
```

Comment:
Ça c’est surprenant. En général on s’attendrait à faire `… --format xml > fichier.xml`.

Par ailleurs, le format que tu génères n’est pas un simple XML, mais suis un format qui ressemble à celui de JUnit. Il devrait s’appeler junit ou junit-xml si c’est le cas.

Reply:
The `extension:filename.xml` syntax for the `--format` flag is something I found frequently in tools that produce output for tests in different formats. That's why I used it here.

But I agree, outputting to  stdout gives more control to the end user.